### DISCUSSION TITLE
[Feature Request] Button to `collapse all`

### DISCUSSION THREAD

**Discussion by tajchert** [AykutSarac/jsoncrack.com#128] [Ideas]
Currently by default all arrays are in an "expanded" state, it would be great to either control this in settings (what is the default behavior) or have a button to "collapse all".

This would greatly improve UX with huge files with dozens of lists that are impossible to navigate in preview windows when everything is expanded by defaults (there is a lot of scrolling otherwise). 

Also thanks all for this tool, it brings (some) joy to working with huge/complex JSONs!

**Comment by Pablohn26** [comment]
I would add that if there is a setting with `collapse all` by default, we may avoid unresponsiveness problems (browser problems) when loading a JSON with thousands of lines.

**Comment by victorbrambati** [comment]
It is better to collapse only nodes that have many children. What do you think?
